contrib/templates/experimental/scala-job/databricks_template_schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "properties": {
+    "project_name": {
+      "type": "string",
+      "description": "Name of project. Do not add spaces",
+      "order": 1
+    },
+    "custom_workspace_host": {
+      "type": "string",
+      "default": "{{workspace_host}}",
+      "description": "Workspace host url",
+      "order": 2
+    },
+    "artifacts_dest_path": {
+      "type": "string",
+      "description": "Destination path in Databricks where the JAR and other artifacts will be stored You can use /Workspace or /Volumes in Dedicated clusters but only /Volumes in Standard clusters.",
+      "order": 3
+    },
+    "existing_cluster_id": {
+      "type": "string",
+      "description": "Enter the cluster id for an existing cluster or leave empty to create a new cluster",
+      "default": "",
+      "order": 4
+    },
+    "cluster_type": {
+      "skip_prompt_if": {
+	"properties": {
+	    "existing_cluster_id": {
+		    "pattern": "^\\S.*$"
+	    }
+	}
+      },
+      "type": "string",
+      "enum": ["Standard", "Dedicated"],
+      "default": "Standard",
+      "description": "Select cluster type: Dedicated or Standard. If Standard, is the JAR allowlisted by the admin for your workspace? (If not, inform admin: https://docs.databricks.com/en/data-governance/unity-catalog/manage-privileges/allowlist.html)",
```

Comment:
Could we just standardize on 'standard' clusters and offer README.md and/or comment guidance on how to change this after the fact? That avoids the setup-time overhead in understanding and making this decision.

Reply:
Re. informing the admin: that advice seems appropriate to give as part of the volume path question.

(Also, if we didn't use a 'standard' cluster then we wouldn't need to ask about a volume path, saving a lot of setup steps for customers.)